lib/Cookie/Cookie.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Aerys\Cookie;
+
+class Cookie {
```

Comment:
This could be useful in amp/artax as well, maybe it would be nice to have an amp/http-message library in the future containing this code.